### PR TITLE
[IMP] sales: reinvoice expenses to customers

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3476731&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### FWP up to `master`

### CODE REVIEW

File: content/applications/sales/sales/invoicing/expense.rst
Comment (jero-odoo) [inline content/applications/sales/sales/invoicing/expense.rst:118]:
```suggestion
If neither of these options are necessary, click :guilabel:`Create Report` to lock in the expense report that was just configured.
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (jero-odoo) [inline content/applications/sales/sales/invoicing/expense.rst:214]:
```suggestion
Payment` banner is in the upper-right corner of the invoice, signifying this invoice is
```

**Review by jero-odoo — CHANGES_REQUESTED** [review]
@ksc-odoo Great work! Let me know if you have any questions 👍

**Comment by ksc-odoo** [comment]
Thank you for the helpful feedback @jero-odoo -- I made all the suggested edits and, once I get your approval, I'll be sure to tag the Materials team for a 2nd peer review on this doc. Thanks again 👍

**Review by jero-odoo — APPROVED** [review]

**Comment by ksc-odoo** [comment]
Thanks @jero-odoo -- Okay, Materials Team...this one's ready for a second round of Peer Review! (@hojo-odoo @Felicious @brse) 👍

File: content/applications/sales/sales/invoicing/expense.rst
Comment (Felicious) [inline content/applications/sales/sales/invoicing/expense.rst:131]:
```suggestion
the expense, and if there are no issues, they will click the :guilabel:`Approve` button — which
```

**Review by Felicious — APPROVED** [review]
Hi @ksc-odoo ! This doc is really well-written! You did a really amazing and thorough job with this, and all the images really make this workflow easy to follow!

I just had a few observation and optional suggestions on brevity. After looking over those, this doc is *definitely* ready for final review!

File: content/applications/sales/sales/invoicing/expense.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/invoicing/expense.rst:62]: I appreciate you pointing this out, but I just checked a 16.4 runbot, and it has "view all" -- so I think I'm going to leave it as-is, assuming the average user reading this will have the most updated version of Odoo 16. However, if you think it would be smart to change it to adhere to 16.3, instead of 16.4, just let me know and I'll make the necessary change right away.

File: content/applications/sales/sales/invoicing/expense.rst
Comment (ksc-odoo) [reply content/applications/sales/sales/invoicing/expense.rst:97]: Great catch! I made the decision to add a note admonition to the beginning of the workflow to avoid any confusion. Thanks again for mentioning this!

**Comment by ksc-odoo** [comment]
Thank you for the always-helpful feedback, @Felicious I really appreciate the constant attention to detail :)

I made all those suggested changes, and believe this is ready for Final Review @odoo/us-doc-review  👍

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:17]: No need to capitalize _main Odoo dashboard_.

And just a note, in other PRs you've written this as "Odoo main dashboard", I think both are OK, just wanted to point this out if there is a reason behind it 🙂
```suggestion
To install the *Expenses* application, navigate to :menuselection:`main Odoo dashboard --> Apps`,
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:29]: Early break
```suggestion
Then, add a :guilabel:`Customer`, and add a product to the :guilabel:`Order Lines` tab, by clicking
:guilabel:`Add a product`. Next, select a product from the drop-down menu.
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:58]: Escape ampersand
```suggestion
- :guilabel:`Travel \& Accommodation`: any travel or accommodation costs related to a project/order.
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:70]:
```suggestion
:guilabel:`Category` for this example is :guilabel:`[TRANS \& ACC] Travel \& Accommodation`.
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:164]:
```suggestion
To do that, navigate to :menuselection:`main Odoo dashboard --> Sales app`, and select the
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:194]:
```suggestion
To send the invoice to the customer, click :guilabel:`Send \& Print`. Doing so reveals a
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:198]:
```suggestion
Once ready, click :guilabel:`Send \& Print` to send the invoice to the customer. When clicked, the
```

File: content/applications/sales/sales/invoicing/expense.rst
Comment (samueljlieber) [inline content/applications/sales/sales/invoicing/expense.rst:228]:
```suggestion
   - :doc:`invoicing_policy`
   - :doc:`time_materials`
   - :doc:`milestone`
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Nice job with this PR and the improvements to this doc. I have just a couple of small changes that I will push up to move this PR forward, please see below. Thank you!

**Review by samueljlieber — APPROVED** [review]
Hi @StraubCreative this PR is good to go 👍

### FWP up to `master`

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**
